docs/achtergronddocumentatie/artefacten/7554.md

```diff
@@ -1,21 +1,117 @@
 ---
 layout: page-with-side-nav
 title: F7554 - Registreer partij
-date: 28-03-2022
+date: 11-12-2023
 ---
 
 # F7554 - Registreer partij
 
+Hoe de partij wordt geregistreerd hangt samen met de wijze van het toekennen van de ID's. 
+Dit vraagstuk staat beschreven in [V6577](./6577.md)
+
+Vooralsnog gaan we ervan uit dat de ID wordt toegekend door de provider. 
+
 ## Functionaliteit
 
-De functie registreert een nieuwe `partij` inclusief de bijbehorende gerelateerde objecten/groepen:
-- `Digitaal adres`
-- `Externe identificatie`
+De functie registreert een nieuwe `partij` en de daarbij behorende `Partijidentificator`. 
```

Comment:
^zie `partij`, `Partijidentificator` maar ook `partij-identificator`. Hanteren we hier nu steeds de naamgevingsconventies uit het CIM of die uit het uitwisselingsmodel (en dus de API-spec)? De tabel benoemt Objecttypes (CIM), maar verwijst wel naar de API-term `request body`

Reply:
Ik stel voor om de terminologie uit het uitwisselmodel te gebruiken. Om de functies goed te kunnen beschrijven heb ik een aantal technische aspecten nodig. Functies puur functioneel beschrijven biedt niet de duidelijkheid die developers nodig hebben bij het implementeren naar mijn idee. Ik heb daar best wel wat mee geworsteld omdat ik wel begiinen bet met beschrijven in termen van het SIM.... 
Ik moet dus nog wel het consistente gebruik van namen nakijken.